utils.sh

```diff
@@ -10,6 +10,7 @@ export wgbconf="$user_home/.wgbconf.json"
 export DIRS=("/etc/wireguard")
 export token=false
 export token_uri=""
+procerrorlog=$(mktemp)
 
 
 function log_error(){
```

Comment:
I would rename these functions to print_error, print_log etc to differentiate logging from screen output

Reply:
I disagree, stdout is just a case of a fd.

Reply:
My original idea was to have a different approach between printing to screen (as an interface) and logging to file (for developer analysis), and it could be useful to have functions to separate logging between debug, info, warning and error (like we have in log4j, for example).

Said this, I agree to avoid a proliferation of logging functions, so my suggestion is: 

- create two separate functions, for print (to screen) and log (to file), where I agree that technically speaking stdout is just a specific file descriptor, but I would separate print 

- But, I would just use one only function for logging and printing, passing as argument the desired logging level (debug, info, warning, etc) 

Does it sound good?

Reply:
I admit not having exactly a strong opinion here, so please, go ahead with your ideas.